cms/envs/aws.py

```diff
@@ -81,6 +81,9 @@
 CELERY_DEFAULT_QUEUE = DEFAULT_PRIORITY_QUEUE
 CELERY_DEFAULT_ROUTING_KEY = DEFAULT_PRIORITY_QUEUE
 
+# Queue to use for updating grades due to grading policy change
+POLICY_CHANGE_GRADES_ROUTING_KEY = LOW_PRIORITY_QUEUE
```

Comment:
@efischer19 , can you check this.  If this is triggered from CMS, he needs one of those crossing the streams configurations that routes it from cms to lms.